workflow/scripts/sbs_process/apply_ic_field.py

```diff
@@ -0,0 +1,15 @@
+from lib.shared.illumination_correction import apply_ic_field
+from skimage.io import imread, imsave
+
+# load aligned image data
+aligned_image_data = imread(snakemake.input[0])
+aligned_image_data_0 = aligned_image_data[0]
+
+# load illumination correction field
+ic_field = imread(snakemake.input[1])
+
+# apply illumination correction field
+corrected_image_data = apply_ic_field(aligned_image_data_0, correction=ic_field)
```

Comment:
Shouldn't ic_field be what is used to subset aligned_image_data?

Reply:
Good catch! I forgot to fix this after our Slack convo. `ic_field` should be the correction (this is derived with `calculate_ic_field`) but we should be subsetting `aligned_image_data` with `segmentation_cycle - 1`.

Can you confirm that the changes in https://github.com/cheeseman-lab/brieflow/pull/6/commits/8e27bcf3100ed66f826a2e9342a78e8973c2910b are correct?